resources/groups/resources.ps1

```diff
@@ -260,7 +259,7 @@ try {
 
             # Example when correlationValue is extensionAttribute1
             $ADGroupParams = @{
-                SamAccountName  = $samaccountname
+                SamAccountName  = $groupName
```

Comment:
samaccountname can only be 20 chars. Should we make sure these limitations are enforced?

Reply:
I think the limit for the samaccountname of a group is more than 20 chars

Reply:
Right. its 63 chars for groups: https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc776019(v=ws.10)?redirectedfrom=MSDN

Reply:
What would you recommend? If we put the correlation value in the samaccountname then we shouldn't use extensionattribute in my opinion

Reply:
I will test what will happen if the samaccountname will be longer then 63 char